src/styles/app/47-external-markdown.css

```
@@ -1,159 +1,224 @@
-.externalMarkdownEditor {
-	flex: 1;
-	min-height: 0;
-	overflow: auto;
-	padding: 34px 18px 56px;
-	background: var(--bg-primary);
-}
+.externalMarkdownEditorShell {
+	--readable-content-outer-width: min(
+		100%,
+		calc(
+			var(--editor-readable-content-max-width) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)